src/utils.js

```diff
@@ -136,7 +136,30 @@ export const isPruneTraceOff = () =>
   );
 
 export const isSwitchedOff = () => {
-  return TracerGlobals.getTracerInputs().switchOff;
+  return TracerGlobals.getTracerInputs().switchOff || !isValidAlias();
+};
+
+export const getValidAliasesOrEmptyArray = () =>
+  process.env['VALID_ALIASES'] || [];
+
+export const getHandlerContext = () =>
+  TracerGlobals.getHandlerInputs().context || {};
+
+export const getInvokedArn = () => getHandlerContext().invokedFunctionArn || '';
+export const getInvokedVersion = () =>
+  getHandlerContext().functionVersion || '';
+
+export const getInvokedAliasOrNull = () =>
+  safeExecute(() => {
+    return getInvokedArn().split(':').length >= 8
+      ? getInvokedArn().split(':')[7]
+      : null;
+  })();
+
+export const isValidAlias = () => {
+  const validAliases = getValidAliasesOrEmptyArray();
+  const currentAlias = getInvokedAliasOrNull();
+  return validAliases.length === 0 || validAliases.includes(currentAlias);
```

Comment:
This means that when there isn't `validAliases`, the value will be `true`. Is this the behavior we want?